src/main/java/com/genesys/workspace/models/User.java

```diff
@@ -1,11 +1,13 @@
 package com.genesys.workspace.models;
 
+import java.util.Map;
+
 public class User {
     private int dbid;
     private String employeeId;
     private String agentId;
     private String defaultPlace;
-    private KeyValueCollection userProperties;
+    private Map<String,Object> userProperties;
```

Comment:
Can we translate from Map<String, Object> to KeyValueCollection in the constructor maybe? I'd like to keep KeyValueCollection in use in the public API for convenience.